generated/microsoftgraph/microsoft.graph/v1.0/0.1.10-preview/types.md

```diff
@@ -0,0 +1,370 @@
+* **isEnabled**: bool: Indicates whether the permission is enabled.
+* **value**: string: The value of the permission.
+
+## MicrosoftGraphSamlSingleSignOnSettings
+### Properties
+* **relayState**: string: The relative URI the service provider would redirect to after completion of the single sign-on flow.
+
+## MicrosoftGraphServicePrincipalLockConfiguration
+### Properties
+* **allProperties**: bool: Enables locking all sensitive properties. The sensitive properties are keyCredentials, passwordCredentials, and tokenEncryptionKeyId.
+* **credentialsWithUsageSign**: bool: Locks the keyCredentials and passwordCredentials properties for modification where credential usage type is Sign.
+* **credentialsWithUsageVerify**: bool: Locks the keyCredentials and passwordCredentials properties for modification where credential usage type is Verify. This locks OAuth service principals.
+* **isEnabled**: bool: Enables or disables service principal lock configuration. To allow the sensitive properties to be updated, update this property to false to disable the lock on the service principal.
+* **tokenEncryptionKeyId**: bool: Locks the tokenEncryptionKeyId property for modification on the service principal.
+
+## MicrosoftGraphServiceProvisioningError
+### Properties
+* **createdDateTime**: string: The date and time at which the error occurred.
+* **isResolved**: bool: Indicates whether the error has been attended to.
+* **serviceInstance**: string: Qualified service instance (for example, 'SharePoint/Dublin') that published the service error information.
+
+## MicrosoftGraphSpaApplication
+### Properties
+* **redirectUris**: string[]: Specifies the URLs where user tokens are sent for sign-in, or the redirect URIs where OAuth 2.0 authorization codes and access tokens are sent.
+
+## MicrosoftGraphVerifiedPublisher
+### Properties
+* **addedDateTime**: string: The timestamp when the verified publisher was first added or most recently updated.
+* **displayName**: string: The verified publisher name from the app publisher's Partner Center account.
+* **verifiedPublisherId**: string: The ID of the verified publisher from the app publisher's Partner Center account.
+
+## MicrosoftGraphWebApplication
+### Properties
+* **homePageUrl**: string: Home page or landing page of the application.
+* **implicitGrantSettings**: [MicrosoftGraphImplicitGrantSettings](#microsoftgraphimplicitgrantsettings): Specifies whether this web application can request tokens using the OAuth 2.0 implicit flow.
+* **logoutUrl**: string: Specifies the URL that is used by Microsoft's authorization service to log out a user using front-channel, back-channel or SAML logout protocols.
+* **redirectUris**: string[]: Specifies the URLs where user tokens are sent for sign-in, or the redirect URIs where OAuth 2.0 authorization codes and access tokens are sent.
+* **redirectUriSettings**: [MicrosoftGraphRedirectUriSettings](#microsoftgraphredirecturisettings)[]
+
+## MsgraphRelationship
+### Properties
```

Comment:
Shouldn't these have some descriptions?
Also shouldn't we say that the default (if not set) for relationshipSemantics is "append"